librubyfmt/src/parser_state.rs

```diff
@@ -320,22 +312,21 @@ impl ParserState {
                 None => 0,
             };
 
+            let spaces = self.spaces_after_last_newline;
+            debug!("spaces: {} comments: {:?}", spaces, self.comments_to_insert);
+            let bt = Backtrace::new();
+            debug!("{:?}", bt);
```

Comment:
If we're at the point where we're printing backtraces into the debug logger, we should probably use the `tracing` library instead

Reply:
this was pretty one-off. I'll probably just delete